Schyntax/Internals/IntermediateRepresentation.cs

```diff
@@ -58,6 +58,14 @@ public class IrGroup
         public bool HasDaysOfMonthExcluded => _daysOfMonthExcluded != null && _daysOfMonthExcluded.Count > 0;
         public List<IrIntegerRange> DaysOfMonthExcluded => _daysOfMonthExcluded ?? (_daysOfMonthExcluded = new List<IrIntegerRange>());
 
+        private List<IrIntegerRange> _daysOfYear;
+        public bool HasDaysOfYear => _daysOfYear != null && _daysOfYear.Count > 0;
+        public List<IrIntegerRange> DaysOfYear => _daysOfYear ?? (this._daysOfYear = new List<IrIntegerRange>());
```

Comment:
Yep, I totally get that everyone is used to different styles. Just trying to stay consistent within a given codebase.